backend/controller/contentController.js

```diff
@@ -206,17 +205,20 @@ const httpGetContent = asyncHandler(async (req, res, next) => {
   result.contents = await Content.find(query)
     .skip(startIndex)
     .limit(LIMIT)
-    // TODO: test fail, reverted back.
     .sort(sorting.latestContent || sorting.likesCount || sorting.trending);
 
+  if (!result.contents) {
```

Comment:
Will the middleware not stop the user before this?
If yes, then this error message is not correct

Reply:
will change the message "content not able to Fetch"